frontend/src/components/BackButton/index.js

```diff
@@ -0,0 +1,18 @@
+import { PropTypes } from "prop-types";
+import styles from "./index.module.css";
+
+import BackButtonLogo from "assets/svgs/wheel-background/back-button.svg";
+
+const BackButton = ({ href }) => {
+  return (
+    <a href={href} className={styles.backButton}>
+      <img src={BackButtonLogo} alt="Back button"></img>
```

Comment:
Since this is an SVG in React, let's import them directly as components.

https://create-react-app.dev/docs/adding-images-fonts-and-files#adding-svgs